src/styles/blocks/main-block.scss

```diff
@@ -0,0 +1,25 @@
+.main-block {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+
+  max-width: 944px;
```

Comment:
do not hardcode it - calculate max-width value. Set variables for card width and margins or paddings which you used to calculate this value.

Reply:
Great, thank you. Today I saw Danylo doing it on morning meeting